src/kaenguru.rs

```diff
@@ -6,8 +6,9 @@ use serenity::{
     prelude::*,
 };
 use serenity::utils::Color;
-use crate::kaenguru;
+use crate::config::Bots;
 use crate::kaenguru::euro_to_mark::get_euro;
+use crate::replies::{reply_to, ReplyError};
```

Comment:
Why the full path?

Reply:
Because the project does not compile with `crate::` or `crate::replies::` removed.